inference-engine/tests/functional/plugin/cpu/single_layer_tests/cum_sum.cpp

```diff
@@ -0,0 +1,250 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ngraph_functions/builders.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace CPULayerTestsDefinitions {
+
+using cumSumParams = std::tuple<
+    ngraph::element::Type, // data precision
+    std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>>, // input shape
```

Comment:
make alias for this smt like `CumSumShape`

Reply:
Thanks, done